### PR TITLE
Feature/bootstrapping

### PR DESCRIPTION
**Pull request by poteto**
Get some stuff on page.

### CODE REVIEW

**Comment by poteto** [comment]
Nothing too exciting here so I'm going to go ahead and merge this one in @catkins. Only thing that needs clean up is the test for the `google-map` component, but I'll work on that on another branch

**Comment by catkins** [comment]
LGTM :+1:

File: app/pods/application/adapter.js [hunk 1]
```diff
@@ -4,5 +4,5 @@ import DS from 'ember-data';
 import config from '../../config/environment';
 
 export default DS.FirebaseAdapter.extend({
-  firebase: new Firebase(`https://${config.firebase_instance}.firebaseIO.com`)
+  firebase: new Firebase(`https://${config.firebaseUrl}.firebaseIO.com`)
```
Comment (catkins) [inline app/pods/application/adapter.js:7]: Perhaps would be a little more clear to just have the whole url in the config variable?

File: app/pods/listing/model.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  name        : DS.attr('string'),
+  lat         : DS.attr('number'),
+  lng         : DS.attr('number'),
+  price       : DS.attr('number'),
+  description : DS.attr('string'),
+  state       : DS.belongsTo('state', { async: true }),
+  city        : DS.attr('string')
```
Comment (catkins) [inline app/pods/listing/model.js:10]: I see that you're still using sublime aligntab :stuck_out_tongue:

**Comment by poteto** [comment]
Haha yes :p On a side note, I just realised `state` needs a center `lat` and `lng` for centering the map. I'll open an issue for it

**Pull request merged**